### DISCUSSION TITLE
Leitura do Appsettings

### DISCUSSION THREAD

**Discussion by RanieryCG** [balta-io/2811#8] [Q&A] [answered]
Olá! Eu utilizo a leitura do AppSettings passando o IConfiguration como injeção de dependências direto. 

Está errado? Sempre funcionou para mim, mas não é uma boa prática?

Eu sempre usei IConfiguration _configuration 

e ai _configuration.GetSection("Atributo:Atributo2").Value

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @RanieryCG, como vai?

Não tem certo ou errado... pode ser feito desta forma sim... Particularmente acho que ter uma classe de configuração facilita, mas se você está bem desta forma, tranquilo :)
